src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -203,3 +243,28 @@ def _build_encrypted_message(self, message_dict, loop_start_time):
                 loop_current_time=loop_start_time)
 
         return message
+
+    def _send_mail(self, mime_message, recipients):
+        """Adds From and To headers to the message object, then passes the message to
+        sendmail to be queued by the local MTA.
+
+        mime_message: A MIMEMultipart message object describing the e-mail to send.
+        recipients:     A list of e-mail addresses to send the e-mail to.
+        """
+        self.logger.info('Sending message via sendmail.')
+
+        recipients_string = ', '.join(recipients)
+
+        mime_message['From'] = self.config['sender']['email']
+        mime_message['To'] = recipients_string
+
+        sendmail_process = subprocess.Popen(['sendmail', '-t'], stdin=subprocess.PIPE)
+        sendmail_process.communicate(str(mime_message))
+        sendmail_process.stdin.close()
+        sendmail_process.wait()
+
+        if sendmail_process.returncode >= 64:
+            raise SendmailError('Sendmail returned error code %s' % \
+                    (sendmail_process.returncode))
```

Comment:
I think the inner parenthesis are unnecessary. I'd also put a period after the placeholder.